source/index.txt

```diff
@@ -1,13 +1,104 @@
 :orphan:
 
-=======================================
-The MongoDB |version| Dev Series Manual
-=======================================
+================================================================
+The MongoDB |version| Dev Series Commandline Tools Documentation
```

Comment:
Note to change this to a source constant:

`{+server-utils+}`

And add a new field to the `source_constants` object in `conf.py` ->

`'server-utils' : 'Server Utilities'`

Reply:
Great minds. But Snooty doesn't appear to permit source constants in titles. I've done this everywhere except for the title, which I left hard-coded.